packages/functional-tests/tests/syncV3/fxDesktopHandshake.spec.ts

```diff
@@ -172,22 +204,30 @@ test.describe('severity-2 #smoke', () => {
       await page.goto(
         `${
           target.contentServerUrl
-        }?force_auth&automatedBrowser=true&${query.toString()}`
+        }?force_auth&automatedBrowser=true&context=fx_desktop_v3&service=sync&${query.toString()}`
       );
       await login.respondToWebChannelMessage(eventDetailStatus);
       await login.checkWebChannelMessage('fxaccounts:fxa_status');
       expect(await login.getPrefilledEmail()).toContain(syncEmail);
     });
 
-    test('Sync - user signed into browser, no user signed in locally', async ({
+    test('Non-Sync force_auth page - user signed into browser is different to requested user', async ({
       emails,
       target,
       syncBrowserPages: { login, page },
     }) => {
-      const [email] = emails;
+      await Promise.all(
+        emails.map(async (email) => {
+          target.auth.signUp(email, PASSWORD, {
```

Comment:
Same general half-baked theory as the non-await `Promise.all()` comment above.

Reply:
Thank you! will fix!